packages/common/src/utils/validationUtils.test.ts

```diff
@@ -1027,4 +1027,44 @@ describe('Set initial values to a certificate', () => {
       expect(certificate.data['1.2'].visible).toBe(true)
     })
   })
+
+  describe('Validate highlight rule for boolean value', () => {
```

Comment:
These tests doesn't test the added logic. Instead it tests the parseExpression, which is already verified by other tests that it can evaluate a boolean expression.